src/react-disposable-decorator.js

```diff
@@ -37,4 +37,7 @@ export default function(DecoratedComponent) {
 			this.disposables = [];
 		}
 	}
+
+  Disposable.displayName = `Disposable(${DecoratedComponent.displayName || DecoratedComponent.name})`
```

Comment:
Couldn't you use `static displayName = ...` inside of the class, instead of this?

Since they are equivalent, doesn't matter, just an fyi.

Reply:
Awesome! I forgot about this, I'll use static instead.